packages/hash/api/src/model/util.ts

```diff
@@ -51,4 +59,150 @@ export const RESTRICTED_SHORTNAMES = [
   "v2",
 ];
 
-export const NIL_UUID = "00000000-0000-0000-0000-000000000000";
+export const nilUuid = "00000000-0000-0000-0000-000000000000" as const;
+
+/**
+ * @todo: create workspace types in an account that's dedciated to
+ * the HASH workspace. For now we're just chucking them in the root account.
+ */
+export const workspaceAccountId = nilUuid;
+
+export const worskspaceTypesNamespaceUri =
+  "https://example.com/@workspace/types";
+
+export const blockprotocolTypesNamespaceUri =
+  "https://blockprotocol.org/@blockprotocol/types";
+
+type SchemaKind =
+  | EntityType["kind"]
+  | PropertyType["kind"]
+  | DataType["kind"]
+  | LinkType["kind"];
+
+const schemaKindSlugs: Record<SchemaKind, string> = {
+  entityType: "entity-type",
+  dataType: "data-type",
+  propertyType: "property-type",
+  linkType: "link-type",
+};
+
+const slugifySchemaTitle = (title: string): string =>
+  slugify(title.toLowerCase());
```

Comment:
I don't _think_ it would result in the same string if slugify interprets the case you have. But admittedly I haven't looked at the implementation / docs.

Assuming slugify works on PascalCase stuff for example, e.g. turning `ThisSentence` into `this-sentence` as well as things separated by spaces `This Sentence` into `this-sentence`, then I wonder what would happen if you had `ThisSentence Foo`?

Was thinking maybe if you lowercase first you'd have `thissentence foo` turned into `thissentence-foo` but if you did it after you _might_ have `This-Sentence-Foo` turned into `this-sentence-foo`. But I don't know if slugify would apply rules for space-separated stuff at the same time as PascalCase if it even does support it.

**Anyway** I'm just rambling and I don't think it's important :D 

Thank you for the change